.github/actions/setup-node/action.yaml

```diff
@@ -12,4 +17,4 @@ runs:
         cache-dependency-path: "site/yarn.lock"
     - name: Install node_modules
       shell: bash
-      run: ./scripts/yarn_install.sh
+      run: ./scripts/yarn_install.sh ${{ inputs.yarn-install-flags }}
```

Comment:
I think we need to make sure this input is part of the cache key, otherwise we'll be hopping back-n-forth between prettier-only and full install?